src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -448,21 +448,21 @@ export default class FilterOptionsComponent extends Component {
             if (!filter) {
               filterOption.classList.remove('hiddenSearch');
               filterOption.classList.remove('displaySearch');
-              labelEl.innerHTML = labelText;
+              labelEl.textContent = labelText;
```

Comment:
in case you haven't already, it would be good to spin up this component locally and test that the text still appears as expected. you should be able to just serve and navigate to the acceptance test page for it and try things out

Reply:
The jest test I added already covers what I changed. Ack